src/handycon/handhelds/aya_gen7.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+# This file is part of Handheld Game Console Controller System (HandyGCCS)
+# Copyright 2022-2023 Derek J. Clark <derekjohn.clark@gmail.com>
+
+import sys
+from evdev import InputDevice, InputEvent, UInput, ecodes as e, list_devices, ff
+
+from .. import constants as cons
+
+handycon = None
+
+def init_handheld(handheld_controller):
+    global handycon
+    handycon = handheld_controller
+    handycon.BUTTON_DELAY = 0.10
+    handycon.CAPTURE_CONTROLLER = True
+    handycon.CAPTURE_KEYBOARD = True
+    handycon.CAPTURE_POWER = True
+    handycon.GAMEPAD_ADDRESS = 'usb-0000:c4:00.3-4/input0'
+    handycon.GAMEPAD_NAME = 'Microsoft X-Box 360 pad'
+    handycon.KEYBOARD_ADDRESS = 'isa0060/serio0/input0'
+    handycon.KEYBOARD_NAME = 'AT Translated Set 2 keyboard'
+
+
+# Captures keyboard events and translates them to virtual device events.
+async def process_event(seed_event, active_keys):
+    global handycon
+
+    # Button map shortcuts for easy reference.
+    button2 = handycon.button_map["button2"]  # Default QAM
+    button5 = handycon.button_map["button5"]  # Default MODE
+
+    ## Loop variables
+    button_on = seed_event.value
+
+    # Automatically pass default keycodes we dont intend to replace.
+    if seed_event.code in [e.KEY_VOLUMEDOWN, e.KEY_VOLUMEUP]:
+        await handycon.emit_events([seed_event])
+
+    # BUTTON 2 (Default: QAM) Small Button
+    if active_keys in [[32, 125], [125]] and button_on == 1 and button2 not in handycon.event_queue:
```

Comment:
Why are we checking a list of lists here? The other 4 keys *should* be producing a KEY_LEFTMETA(125) if they are working properly. I think we have some scancode issues or a bugged IRQ.